doc/Makefile.am

```diff
@@ -61,5 +64,23 @@ cryptoapiref: mkapiref.py \
 html-local: apiref cryptoapiref
 	@$(SPHINXBUILD) -M html "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
+sphinx-build: apiref cryptoapiref
+	@$(SPHINXBUILD) -M $(SPHINX_BUILDER) "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+man: apiref cryptoapiref
+	$(MAKE) $(MAKEOPTS) sphinx-build SPHINX_BUILDER=man
+
+epub: apiref cryptoapiref
+	$(MAKE) $(MAKEOPTS) sphinx-build SPHINX_BUILDER=epub
+
+pdf-local: apiref cryptoapiref
+	$(MAKE) $(MAKEOPTS) sphinx-build SPHINX_BUILDER=latexpdf
+
+latexpdf: pdf
+
+doc:
+	for SPHINX_BUILDER in $(SPHINX_BUILDERS); do \
+		$(MAKE) $(MAKEOPTS) sphinx-build SPHINX_BUILDER=$${SPHINX_BUILDER}; \
+	done
```

Comment:
It would be better to run command directly rather than recursively calling make:

```make
man: apiref cryptoapiref
	@$(SPHINXBUILD) -M man "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)

epub: apiref cryptoapiref
	@$(SPHINXBUILD) -M epub "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)

pdf-local: apiref cryptoapiref
	@$(SPHINXBUILD) -M latexpdf "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)

doc: html man epub
```

Reply:
Yes, it might. But that is a repeated copy & pasting rules for all specific targets, where only a minimal change occurs. I tried to make it reusing common sphinx-build target, though I am not sure how to make it nicer. If you prefer it copied separately, that is of course possible.

Reply:
The current proposed way is also kind of copy-and-paste.
Is there any real usecase or demand to offer this kind of extra documentation format like epub or pdf?
If this is just added for completeness or "maybe useful someday", do not add now and wait when they are actually needed.

Reply:
I added these, because ``make help`` lists them. But when they are tried, they do not work. Perhaps make help might be removed, if only make html is supported.